symfony/messenger/4.3/post-install.txt

```diff
@@ -0,0 +1,19 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * You're ready to use the Messenger component. You can define your own message buses
+    or start using the default one right now by injecting the <info>messenger.bus.default</info> service
+    or typehinting <info>Symfony\Component\Messenger\MessageBusInterface</info> in your code.
+
+  * If you need to send messages to your broker, you can benefit from the built-in
+    AMQP transport by:
+
+    1. Installing the AMQP pack by running <info>composer require amqp</>;
+    2. Uncommenting the <comment>MESSENGER_TRANSPORT_DSN</> env var
+       and <comment>framework.messenger.transports.amqp</> config;
```

Comment:
> Uncomment the <comment>MESSENGER_TRANSPORT_DSN</> env var in <comment>.env</comment>
       and <comment>framework.messenger.transports.amqp</> in <comment>config/packages/messenger.yaml</comment>